src/chi/components/sidebar/sidenav.scss

```diff
@@ -55,39 +54,161 @@ $sidenav-lg-width: 12.5rem;
         line-height: 1rem;
         text-align: center;
       }
-
-      &:hover {
-        color: $primary-color;
-      }
     }
 
     &.-active {
       > a {
+        background-color: rgba(set-color(blue, 70), 0.08);
         box-shadow: inset 0.125rem 0 0 $primary-color;
         color: $primary-color;
         font-weight: 600;
+
+        &.-unselected {
+          background-color: transparent;
+          box-shadow: none;
+        }
       }
     }
 
-    &.-active > a { // List item -active. The one that has the left 3px blue border.
-      background-color: set-color(grey, 20);
+    &:not(.-active) {
+      a.-active {
+        background-color: set-color(grey, 10);
+        box-shadow: inset 0.125rem 0 0 set-color(grey, 100);
+      }
     }
 
     & > a {
       &.-active { // Drawer shown, will sets active the trigger (this anchor element)
-        background-color: set-color(grey, 20);
+        background-color: set-color(grey, 10);
```

Comment:
This line can be removed to ensure consistent active behavior with inverse/global nav. In the inverse version, the active item bg stays green when its submenu drawer is triggered. This line is causing the base versions active item bg to change from blue to grey when its submenu drawer is triggered.